src/Illuminate/Database/DatabaseServiceProvider.php

```diff
@@ -60,7 +60,7 @@ public function register()
     protected function registerEloquentFactory()
     {
         $this->app->singleton(FakerGenerator::class, function () {
-            return FakerFactory::create();
+        	return FakerFactory::create(config('database.faker_locale', 'en_US'));
```

Comment:
you can't use the config helper function from a service provider not dependent on the foundation helpers